recipe/meta.yaml

```diff
@@ -1,13 +1,14 @@
 {% set name = "nbclassic" %}
-{% set version = "0.5.3" %}
+{% set version = "0.5.4" %}
+{% set sha256 = "312b3f7d7ff2e6c261d51799bb12e6493498ab47d3469d3a01015d5533fd4d2b" %}
```

Comment:
There is no need for the jinja2 variable if it's used only once. The recipe now has two lines about sha256 instead of one line

Reply:
I've taken this as an action in "future-proofing"/standardizing as I've seen several instances where the variable was defined only refer to the variable once and do this. I assumed this was a preferred mechanism that we were all moving towards.

If it is something people have strong opinions on, maybe the linter needs a new rule(?).